assignment/assignment-tool/tool/src/webapp/js/assignments.js

```diff
@@ -964,23 +964,24 @@ ASN.handleReportsTriangleDisclosure = function (header, content)
     }
 }
 
-ASN.displayCopyLink = function(assignmentCount)
+ASN.displayCopyLink = function(assignmentCount, directUrl)
 {
+    $('.' + assignmentCount + '_urlholder').attr("value", ASN.toggleShortUrlOutput(directUrl, $('#directToolUrl'), assignmentCount + '_urlholder'));
```

Comment:
You have multiple elements in the page that have `#directToolUrl` when you should only have one element on the page with that ID.
`ASN.toggleShortUrlOutput()` doesn't look to return anything so why are you setting the value of an element to it?

Reply:
Changes made.